test/Removal.t.sol

```diff
@@ -272,6 +267,143 @@ contract Removal_release_listed is UpgradeableMarket {
   }
 }
 
+contract Removal_release_unlisted_listed_and_retired is UpgradeableMarket {
+  uint256[] private _removalIds;
+  address[] private _expectedOwners;
+  uint256[] private _expectedBalances = [
+    uint256(0),
+    uint256(0),
+    uint256(0.1 ether)
+  ];
+
+  function setUp() external {
+    _expectedOwners = [
+      _namedAccounts.supplier,
+      address(_market),
+      address(_certificate)
+    ];
+    _removalIds = _seedRemovals({
+      to: _namedAccounts.supplier,
+      count: 1,
+      list: false
+    });
+    _removal.safeBatchTransferFrom({
+      from: _namedAccounts.supplier,
+      to: address(_market),
+      ids: new uint256[](1).fill(_removalIds[0]),
+      amounts: new uint256[](1).fill(0.5 ether),
+      data: ""
+    });
+    assertEq(
+      _removal.balanceOf(_namedAccounts.supplier, _removalIds[0]),
+      0.5 ether
+    );
+    assertEq(_removal.balanceOf(address(_market), _removalIds[0]), 0.5 ether);
+    uint256 ownerPrivateKey = 0xA11CE;
+    address owner = vm.addr(ownerPrivateKey);
+    uint256 checkoutTotal = _market.getCheckoutTotal(0.25 ether);
+    vm.prank(_namedAccounts.admin);
+    _bpNori.deposit(owner, abi.encode(checkoutTotal));
+    SignedPermit memory signedPermit = _signatureUtils.generatePermit(
+      ownerPrivateKey,
+      address(_market),
+      checkoutTotal,
+      1 days,
+      _bpNori
+    );
+    vm.prank(owner);
+    _market.swap(
+      owner,
+      checkoutTotal,
+      signedPermit.permit.deadline,
+      signedPermit.v,
+      signedPermit.r,
+      signedPermit.s
+    );
+    assertEq(_certificate.balanceOfRemoval(0, _removalIds[0]), 0.25 ether);
+  }
+
+  function test() external {
+    _removal.release(_removalIds[0], 0.9 ether);
+    assertEq(
+      _removal.balanceOfBatch(
+        _expectedOwners,
+        new uint256[](3).fill(_removalIds[0])
+      ),
+      _expectedBalances
+    );
+  }
+}
+
+/** @dev A test that asserts the ability to release a removal retired across 2 certificates */
+contract Removal_release_unlisted_listed_and_retired_2x is UpgradeableMarket {
```

Comment:
```suggestion
contract Removal_release_retired_2x is UpgradeableMarket {
```

Reply:
I prefer the original as it's more descriptive and follows the name of the non-2x version